probe-rs-tools/Cargo.toml

```diff
@@ -14,6 +14,17 @@ description = "A collection of on chip debugging tools to communicate with micro
 
 default-run = "probe-rs"
 
+[features]
+default = []
+# Include server/client functionality
+remote = [
+    "tokio-tungstenite",
```

Comment:
I think these days dependency enabling featureflags should be prefixed with `dep:`

Reply:
It doesn't really matter, the difference is whether an implicit feature exists for `tokio-tungstenite`. I can prefix, and that removes the implicit feature